apps/web/public/static/locales/en/common.json

```diff
@@ -4095,8 +4095,25 @@
   "calendar_view": "Calendar view",
   "booking_history": "Booking History",
   "booking_history_description": "View the history of actions performed on this booking",
-  "audit_action": {
-    "created": "Created"
+  "booking_audit_action": {
+    "created": "Created",
+    "cancelled": "Cancelled",
+    "rescheduled": "Rescheduled {{oldDate}} -> <0>{{newDate}}</0>",
+    "rescheduled_from": "Rescheduled <0>{{oldDate}}</0> -> {{newDate}}",
+    "accepted": "Accepted",
+    "reschedule_requested": "Reschedule Requested",
+    "attendee_added": "Attendee Added",
+    "host_no_show_updated": "Host No-Show Updated",
+    "rejected": "Rejected",
+    "attendee_removed": "Attendee Removed",
+    "reassignment": "Reassignment",
+    "booking_reassigned_to_host": "Booking reassigned to {{host}}",
```

Comment:
booking_reassigned_to_host already exists with value "Booking reassigned to {{host}}"